test_unstructured_ingest/dest/azure-cognitive-search.sh

```diff
@@ -28,13 +28,13 @@ source "$SCRIPT_DIR"/cleanup.sh
 function cleanup {
   # Index cleanup
   response_code=$(curl -s -o /dev/null -w "%{http_code}" \
-    "https://utic-test-ingest-fixtures.search.windows.net/indexes/$DESTINATION_INDEX?api-version=$API_VERSION" \
+    "https://ingest-test-azure-cognitive-search.search.windows.net/indexes/$DESTINATION_INDEX?api-version=$API_VERSION" \
```

Comment:
nit: should we make a variable at the top? (based on re-use)

Reply:
Can do, on it